app/logic/courseManagement.py

```diff
@@ -14,26 +15,36 @@ def unapprovedCourses(termId):
     Queries the database to get all the neccessary information for submitted courses.
     '''
 
-    return (Course.select(Course, Term)
+    unapprovedCourses = (Course.select(Course, Term, CourseStatus, fn.GROUP_CONCAT(" " ,User.firstName, " ", User.lastName).alias('instructors'))
+                  .join(CourseInstructor, JOIN.LEFT_OUTER)
+                  .join(User, JOIN.LEFT_OUTER)
+                  .switch(Course)
                   .join(CourseStatus)
                   .switch(Course)
                   .join(Term)
```

Comment:
I think it's a clearer query if we hide the `.switch()` calls a little. Go ahead and leave them at the end of their line instead of breaking to a new one